From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Juro Bystricky <juro.bystricky@intel.com>
Date: Fri, 27 Mar 2020 18:43:56 -0700
Subject: [PATCH] cflags

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4122a22..9811546 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1465,9 +1465,9 @@ if(WALL AND NOT MSVC)
        Geo/*.cpp Graphics/*.cpp Mesh/*.cpp Numeric/*.cpp Parser/*.cpp
        Plugin/*.cpp Post/*.cpp Solver/*.cpp)
   set(WF "-Wall")
-  check_cxx_compiler_flag("-Wint-to-void-pointer-cast" WCAST)
+  check_cxx_compiler_flag("-Wint-to-pointer-cast" WCAST)
   if(WCAST)
-    set(WF "${WF} -Wno-int-to-void-pointer-cast")
+    set(WF "${WF} -Wno-int-to-pointer-cast")
   endif()
   check_cxx_compiler_flag("-Wdeprecated-declarations" WDEPREC)
   if(WDEPREC)
