diff -rupN gmsh-4.4.1-source/CMakeLists.txt gmsh-4.4.1-source-new/CMakeLists.txt
--- gmsh-4.4.1-source/CMakeLists.txt	2019-07-29 16:44:32.383600789 +0200
+++ gmsh-4.4.1-source-new/CMakeLists.txt	2019-07-29 16:44:32.385600723 +0200
@@ -1838,7 +1838,7 @@ endif()
 if(ENABLE_BUILD_SHARED OR ENABLE_BUILD_DYNAMIC)
   install(TARGETS shared DESTINATION ${GMSH_LIB} OPTIONAL)
   # FIXME once we cleanup the installation of the python module
-  install(TARGETS shared DESTINATION ${CMAKE_CURRENT_SOURCE_DIR}/api OPTIONAL)
+  # install(TARGETS shared DESTINATION ${CMAKE_CURRENT_SOURCE_DIR}/api OPTIONAL)
 endif()
 
 if(ENABLE_ONELAB)
